pkgs/tools/package-management/nix-serve/default.nix

```diff
@@ -12,23 +21,34 @@ stdenv.mkDerivation rec {
     sha256 = "0mjzsiknln3isdri9004wwjjjpak5fj8ncizyncf5jv7g4m4q1pj";
   };
 
-  buildInputs = [ perl nix ]
-    ++ (with perlPackages; [ DBI DBDSQLite Plack Starman ]);
+  buildInputs = [ makeWrapper ];
 
-  dontBuild = false;
+  propagatedBuildInputs = [
+    nix
+    perl
+    perlPackages.DBDSQLite
+    perlPackages.DBI
+    perlPackages.Plack
+    perlPackages.Starman
+  ];
+
+  phases = ["unpackPhase" "installPhase"];
```

Comment:
The previous version specifically skipped the `buildPhase`; I'm not sure if this is still necessary but at the very least naming explicit phases reduces the output (i.e. there isn't any "no configure script found" messages etc).